teps/0131-tekton-conformance-policy.md

```diff
@@ -0,0 +1,251 @@
+For **Tekton users** who use Tekton conformant primitives to define CI/CD pipelines, the benefits are obvious:
+- **Convenience**: users don't have to manage their own Tekton services.
+- **Interoperability**: users are able to move conformant primitives between vendors, thereby  avoiding vendor lock-in.
+
+
+For **Tekton vendors** who provide a service that conforms to this Policy, they can:
+* **Leverage community effort**:  leverage ecosystem-driven tasks and pipelines, docs, community support via slack channels. For vendors who build the product upon community Tekton, they can leverage the new features, bug fixes, performance improvements etc.
+* **Take advantage of the community's success**: Tekton users can bring their conformant primitives (`v1.PipelineRun`, `v1.TaskRun`, `v1.Pipeline`, `v1.Task` in the current policy version) to the vendor's platform.
+* **Have portability guarantees** for their users.
+
+### Goals
+
+* Defines a **detailed conformance policy** that vendors can claim conformance to on the service they provide.
+* Defines a **versioning story** on the Conformance Policy to enable us make updates on the policy.
+* Defines a **policy update procedure**.
+* Defines a **way to enforce the policy**.
+
+### Non-Goals
+
+* **HTTP API Conformance**: built upon the current conformance policy, may require vendors to support certain HTTP verbs, expose HTTP API Endpoints that conform to a specific format, support a certain format of request body and response body, support certain response codes etc.
+
+
+## Glossary
+
+**Tekton Primitives**: are a set of basic data types in Tekton, providing common abstractions for describing and executing container-based, run-to-completion workflows, typically in service of CI/CD scenarios.
+* **Pipelines Primitives**: defines workloads execution pipeline
+  * Task, TaskRun, Pipeline, PipelineRun, CustomRun.
+* **Triggers Primitives**: defines events listening and handling
+  * Triggers, TriggerBindings.
+* **Results Primitives**: defines execution history of workloads defined by Pipeline Primitives.
+  * Results, Records
+
+**Tekton API Specification**: describes ***Tekton Primitives*** in detail.
+**Tekton GA API Specifications**: describes ***GA (V1 for now) Tekton Primitives*** in detail.
+
+**Tekton Pipelines API Specification**: describes ***Tekton Pipelines Primitives*** (`Task`, `TaskRun`, `Pipeline`, `PipelineRun`, `CustomRun` of all version tracks) in detail.
+**Tekton GA Pipelines API Specification**: describes ***Tekton GA Pipelines Primitives*** (`v1.Task`, `v1.TaskRun`, `v1.Pipeline`, `v1.PipelineRun`) in detail.
+
+**Tekton Conformance API Specification**: ***core Tekton GA API Specification*** each vendor MUST implement.
+**Tekton Pipelines Conformance API Specification**: ***core Tekton GA Pipelines API Specification*** each vendor MUST implement: `v1.TaskRun`, `v1.PipelineRun` with only core fields included, within where `v1.Task` and `v1.Pipeline` can be incorporated by reference.
```

Comment:
I'm not sure what "with only core fields included" means

Reply:
Updated the definition, core fields mean all REQUIRED fields for conformance that every vendor has to support. Hopefully it's clearer!